api/shared-modules/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@iota/is-shared-modules",
-  "version": "0.1.11",
+  "version": "0.1.12",
```

Comment:
I just published version 0.1.12 from my branch 62-fix-dependencies to have it available for the updates of the ssi-bridge and audit-trail. It would be nice if you could wait with this PR for my PR to update the dependencies.